src/vlinder/trbs.py

```diff
@@ -41,6 +43,8 @@ def __init__(self, name, file_path=None, file_extension=None):
         self.output_dict = {}
         self.visualizer = None
         self.exporter = None
+        self.report = None
```

Comment:
het wordt nu in de code steeds anders genoemd: report, presentation, powerpoint
Kunnen we 1 kiezen zodat het duidelijk is in alle files dat het om hetzelfde ding gaat?